apps/tailwind-components/components/input/Ontology.vue

```diff
@@ -306,17 +323,17 @@ async function updateSearch(value: string) {
           {{ valueLabels[name] }}
         </Button>
       </div>
-      <div class="flex flex-wrap gap-2 mb-2">
+      <div class="flex flex-wrap gap-2">
```

Comment:
There is some slight content shifting here. I think adding `items-center justify-start` will fix the vertical alignment.

Reply:
Great. Looks good.